topics/proteomics/tutorials/neoantigen-1a-fusion-database-generation/workflows/index.md

```diff
@@ -0,0 +1,9 @@
+---
+redirect_from:
+  - /topics/proteomics/tutorials/neoantigen-1-fusion-database-generation/workflows/main_workflow.html
```

Comment:
This line should not be here.

This index.md file create the page with the table showing all workflows for the tutorial ([example](https://training.galaxyproject.org/training-material/topics/proteomics/tutorials/proteogenomics-dbcreation/workflows/)). But we want this old URL of the workflow itself to redirect to the new URL for the workflow, not the overview page.

However, because we cannot add metadata to the workflow file itself, this is a bit trickier. For the workflows, you should create a file in this folder: https://github.com/galaxyproject/training-material/tree/main/redirects, one per redirected workflow, with the old and new URL in `redirect_from` and `redirect_to`